src/utils/types.ts

```diff
@@ -0,0 +1,46 @@
+export type Character = {
```

Comment:
It's ok to define your own types, but bear in mind that the typing is already defined in the API so probably you can use an extra tool like codegen to generate, hooks based on your grapqhl queries/mutations and the needed types based on the API schema